tests/python/pants_test/rules/test_test_integration.py

```diff
@@ -104,36 +104,59 @@ def test_fail():
         >     assert False
         E     assert False
 
-        testprojects/tests/python/pants/dummies/test_fail.py:2: AssertionError
+        pants/dummies/test_fail.py:2: AssertionError
         =========================== 1 failed in SOME_TEXT ===========================
 
 
         testprojects/tests/python/pants/dummies:failing_target                          .....   FAILURE
         """),
     )
 
-  def test_source_dep(self):
+  def test_source_dep_absolute_import(self):
     pants_run = self.run_passing_pants_test([
-      'testprojects/tests/python/pants/dummies:target_with_source_dep',
+      'testprojects/tests/python/pants/dummies:target_with_source_dep_absolute_import',
     ])
     self.assert_fuzzy_string_match(
       pants_run.stdout_data,
       dedent("""\
-        testprojects/tests/python/pants/dummies:target_with_source_dep stdout:
+        testprojects/tests/python/pants/dummies:target_with_source_dep_absolute_import stdout:
         ============================= test session starts ==============================
         platform SOME_TEXT
         rootdir: SOME_TEXT
         plugins: SOME_TEXT
         collected 1 item
 
-        testprojects/tests/python/pants/dummies/test_with_source_dep.py .        [100%]
+        pants/dummies/test_with_source_dep_absolute_import.py .                  [100%]
 
         =========================== 1 passed in SOME_TEXT ===========================
 
 
-        testprojects/tests/python/pants/dummies:target_with_source_dep                  .....   SUCCESS
+        testprojects/tests/python/pants/dummies:target_with_source_dep_absolute_import  .....   SUCCESS
         """)
-    )
+      )
+
+  def test_source_dep_relative_import(self):
+    pants_run = self.run_passing_pants_test([
+      'testprojects/tests/python/pants/dummies:target_with_source_dep_relative_import',
+    ])
+    self.assert_fuzzy_string_match(
+      pants_run.stdout_data,
+      dedent("""\
+        testprojects/tests/python/pants/dummies:target_with_source_dep_relative_import stdout:
+        ============================= test session starts ==============================
+        platform SOME_TEXT
+        rootdir: SOME_TEXT
+        plugins: SOME_TEXT
+        collected 1 item
+
+        pants/dummies/test_with_source_dep_relative_import.py .                  [100%]
+
+        =========================== 1 passed in SOME_TEXT ===========================
+
+
+        testprojects/tests/python/pants/dummies:target_with_source_dep_relative_import  .....   SUCCESS
+        """)
```

Comment:
I personally prefer keeping parentheses joined here instead of spreading them over a new line, but I think the style is inconsistent all over the repo.

Reply:
Fair. For now, this is using the file convention which I'm going to stick with.

This is precisely why I want to add Black to our codebase - either style works just fine, but the issue is that this statement is unfortunately true :/

> I think the style is inconsistent all over the repo.